eventbridge_api_destination_rule/main.tf

```diff
@@ -30,7 +30,25 @@ resource "aws_cloudwatch_event_target" "target" {
   input          = var.input
   input_path     = var.input_path
   dynamic "input_transformer" {
-    for_each = var.input_transformer == null ? [] : [var.input_transformer]
+    for_each = var.api_destination_input_transformer == null ? [] : [var.api_destination_input_transformer]
+    content {
```

Comment:
Since this is used twice, is there a way to save it to a variable and do you think it's worth it?

Reply:
How do you mean? It is a variable already. Unless you mean moving `var.api_destination_input_transformer == null ? [] : [var.api_destination_input_transformer]` to a variable but that doesn't seem worth it. It's better to have it here so we can see the logic.

Reply:
I meant this block
```
    content {
      input_template = input_transformer.value.input_template
      input_paths    = input_transformer.value.input_paths
    }
```
I think it's just my unfamiliarity with the syntax; I was thinking for each time it loops, it generates the block above so I was thinking of saving it to a variable because in my mind, it's like an object being generated each time and also, the fact that the same block appears below. Initially I was going to ask if we could save the whole `dynamic "input_transformer"` block to some type of function but I don't think terraform has that.